process-cli/src/main/java/org/sonarsource/sonarlint/core/rpc/impl/SonarLintCoreProcess.java

```diff
@@ -19,36 +19,63 @@
  */
 package org.sonarsource.sonarlint.core.rpc.impl;
 
+import java.io.BufferedOutputStream;
 import java.io.ByteArrayInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
 import java.util.concurrent.Callable;
 import java.util.concurrent.CancellationException;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
 import picocli.CommandLine;
 
 @CommandLine.Command(name = "slcore", mixinStandardHelpOptions = true, description = "The SonarLint Core backend")
 public class SonarLintCoreProcess implements Callable<Integer> {
 
+  //FIXME remove the logger before merging
+  private static Logger logger;
+
   @Override
-  public Integer call() {
+  public Integer call() throws FileNotFoundException {
+    logger.info("Starting SonarLint Core backend call()...");
     var originalStdIn = System.in;
     var originalStdOut = System.out;
     System.setIn(new ByteArrayInputStream(new byte[0]));
     // Redirect all logs to stderr for now, would be better to go to a file later
-    System.setOut(System.err);
 
+    System.setErr(new PrintStream(new BufferedOutputStream(new FileOutputStream("SystemErrOoutput.txt"))));
```

Comment:
Was it only for testing? I don't think we agreed on logging to a file for now. And if we do, we should let the client decide of the location (it could be a command line argument)

Reply:
Addressed in [55c408e](https://github.com/SonarSource/sonarlint-core/pull/765/commits/55c408e7ff6269bcbb74e6eb1605a618f60f3987)